src/core/trulens/core/database/legacy/migration.py

```diff
@@ -1,6 +1,6 @@
 """
 This is pre-sqlalchemy db migration. This file should not need changes. It is
-here for backwards compatibility of oldest trulens-eval versions.
+here for backwards compatibility of oldest TruLens versions.
```

Comment:
Do we have any json records in the database, that includes trulens objects? I would guess not, but if there are we would be backwards incompatible with data as well cc: @sfc-gh-pmardziel